CMakeLists.txt

```diff
@@ -205,21 +206,25 @@ EndIf(FAIRSOFT_EXTERN)
 if(NOT DEFINED Boost_NO_SYSTEM_PATHS)
   Set(Boost_NO_SYSTEM_PATHS TRUE)
 endif()
-Set(Boost_NO_BOOST_CMAKE TRUE)
+#Set(Boost_NO_BOOST_CMAKE TRUE)
 #Message("-- Looking for Boost ...")
 # If an older version of boost is found both of the variables below are
 # cached and in a second cmake run, a good boost version is found even
 # if the version is to old.
 # To overcome this problem both variables are cleared before checking
 # for boost.
-Unset(Boost_INCLUDE_DIR CACHE)
-Unset(Boost_LIBRARY_DIRS CACHE)
+#Unset(Boost_INCLUDE_DIR CACHE)
+#Unset(Boost_LIBRARY_DIRS CACHE)
 
 find_package2(PUBLIC Boost
   VERSION 1.67 ${FairMQ_Boost_VERSION}
-  COMPONENTS thread system timer program_options random filesystem chrono exception regex serialization log log_setup atomic date_time signals ${FairMQ_Boost_COMPONENTS}
+  COMPONENTS thread system timer program_options random filesystem chrono exception regex serialization log log_setup atomic date_time ${FairMQ_Boost_COMPONENTS}
 )
 
+if (NOT Boost_INCLUDE_DIRS)
+  get_target_property(Boost_INCLUDE_DIRS Boost::serialization INTERFACE_INCLUDE_DIRECTORIES)
+  message(FATAL "${Boost_INCLUDE_DIRS}")
```

Comment:
This obviously is spurious but you get the idea...

Reply:
Yup, this should not hurt though, so no objections here.

Reply:
FYI, with [CMake 3.15.0-rc2](https://gitlab.kitware.com/cmake/cmake/merge_requests/3438) and later this is addressed upstream.

Reply:
if the intent was to break then that should be message(FATAL_ERROR XXX) as message(FATAL XXX) just printouts FATALXXX

Reply:
@aphecetche yes, that was the intent, but I should have not committed it in any case, so it's fine...

Reply:
@dennisklein Yeah, but I am stuck to 3.13...